docs/index.rst

```diff
@@ -4,15 +4,14 @@
 The CrateDB Admin UI
 ====================
 
-CrateDB ships with a web administration user interface (UI), or *admin UI* for
-short.
+CrateDB ships with a web administration user interface (or *admin UI*).
 
-The CrateDB admin UI runs on every CrateDB node, and you can use it to inspect
+The CrateDB admin UI runs on every CrateDB node. You can use it to inspect
 and interact with the whole CrateDB cluster in a number of ways.
 
 .. SEEALSO::
 
-   The CrateDB admin UI is an open source project and is `hosted on GitHub`_.
+   The CrateDB admin UI is an open-source project `hosted on GitHub`_.
```

Comment:
"open source" should appear without a hyphen as a noun, I think. but with the hyphen as an adjective

I would leave "and is" for consistency reasons, because this is a standard piece of text that appears across all of our documentation projects

however, if you'd like to make a separate issue to track changing this text across all documentation projects, that would be fab!! up to you :)